docs/src/app/components/pages/customization/themes.jsx

```diff
@@ -600,9 +610,7 @@ const ThemesPage = React.createClass({
           using an ES7-style decorator. To start off, define your own raw theme in a JS file like so:
         </p>
 
-        <Paper style={styles.codeExample}>
-          <CodeBlock>{lightRawTheme}</CodeBlock>
-        </Paper>
+        <MarkDownElement text={lightRawTheme} />
 
         <h3 style={styles.title}>1. Using React Lifecycle Methods with Context</h3>
```

Comment:
I was afraid someone was going to say that! :grin: 

I considered it, but as @oliviertassinari pointed out, it needs to be rewritten anyway, so I'll leave it for whoever picks that up.